beater/config/config.go

```diff
@@ -40,18 +40,20 @@ const (
 )
 
 type KibanaConfig struct {
-	Enabled             bool `config:"enabled"`
+	Enabled             bool   `config:"enabled"`
+	APIKey              string `config:"api_key"`
 	kibana.ClientConfig `config:",inline"`
 }
 
 func (k *KibanaConfig) Unpack(cfg *common.Config) error {
```

Comment:
I think we can simplify this method to:

```go
func (k *KibanaConfig) Unpack(cfg *common.Config) error {
        type kibanaConfig KibanaConfig
        if err := cfg.Unpack((*kibanaConfig)(k)); err != nil {
                return err
        }
        k.Enabled = cfg.Enabled()
        k.Host = strings.TrimRight(k.Host, "/")
        return nil
}
```

That way we rely on the config tag for unpacking into APIKey.